images/libvirtd/libvirtd-ds.yaml

```diff
@@ -12,40 +12,21 @@ spec:
     spec:
       hostNetwork: true
       hostPID: true
+      hostIPC: true
       securityContext:
         runAsUser: 0
       containers:
-        - name: daemon
+        - name: libvirtd
           image: kubevirt/libvirtd:latest
           ports:
             - containerPort: 16509
           securityContext:
             privileged: true
             runAsUser: 0
           volumeMounts:
-            - mountPath: /etc/libvirt
-              name: etc
-            - mountPath: /var/run/libvirt
```

Comment:
if libvirt container is resetarted, we would loose all connections to already started qemu processes since all the information in the /var folder is gone. Do you wan't to tackle that later or did you miss it?